tel-search-ch-connector-demo/src_hd/com/axonivy/connector/telsearch/demo/AddressFinder/AddressFinder.xhtml

```diff
@@ -42,14 +46,24 @@
 				</p:panelGrid>
 				<br />
 				<p:orderList value="#{data.entries}" var="entry" responsive="true"
-					style="width:400px" itemValue="#{entry}" converter="ivy.ListItem"
-					id="entryList" controlsLocation="none">
+					itemValue="#{entry}" converter="ivy.ListItem" id="entryList"
+					controlsLocation="none">
 					<f:facet name="caption">Entries</f:facet>
 					<p:column>
 						<pre>#{entry}</pre>
 					</p:column>
 				</p:orderList>
-
+				<br />
+				
+				<style>
+						.ui-orderlist>.ui-g>.ui-g-12 {
+							padding: 0;
+						}
+	
+					.ui-orderlist .ui-orderlist-list {
+						height: 300px !important;
```

Comment:
I assume that was no 'non !important solution' ? -> maybe CSS king @ivy-lli knows better way for re-styling.